cmd/security-agent/app/app.go

```diff
@@ -110,6 +108,13 @@ func start(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return fmt.Errorf("unable to set up global agent configuration: %v", err)
 	}
+
+	// Check if we have at least one component to start based on config
+	if !coreconfig.Datadog.GetBool("compliance_config.enabled") {
```

Comment:
small improvement IMO: do this check after the logger is set up, so that the logged message ends up in the regular logs of the security agent

Reply:
You're right, fixed